app/assets/images/blacklight/geoblacklight-icons.json

```diff
@@ -0,0 +1,2079 @@
+{
```

Comment:
Is this auto generated somehow? 
How would we add new icons?

Is this so they all appear the same color or are there other purposes?

Reply:
This file is auto-generated from IcoMoon. It would help us add additional icons (from Font Awesome or elsewhere) and keep them all color-coordinated.

Try going to IcoMoon.io, click the IcoMoon App button, drag and drop this file onto the screen. You should see all the icons at play here.

Reply:
Ok thanks for that 👍